app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,146 @@
 // Code your JavaScript / jQuery solution here
+var turn = 0; 
+var currentGame = 0; 
+
+WINNING_COMBOS = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6]
+];
+
+$(document).ready(function() { 
+    attachListeners();
+});
+
+function player() {
+    return turn % 2 === 0 ? "X" : "O";
```

Comment:
Kind of a cool JS trick: The int value 0 is falsy. This could read `var player = () => turn % 2 ? 'O' : 'X';`